InfiniteLoopBookProject/src/com/st/lms/model/Entry.java

```diff
@@ -0,0 +1,32 @@
+package com.st.lms.model;
+
+public class Entry {

```

Comment:
Unless you want callers to be able to create an `Entry` object that is neither a `Book`, nor an `Author`, nor a `Publisher`, I suggest making `Entry` `abstract` and its constructors `protected`.